ocp_resources/project.py

```diff
@@ -1,11 +1,11 @@
 from warnings import warn
 
-from ocp_resources.project_project_openshift_io import Project  # noqa: F401
+from ocp_resources.project_project_openshift_io import Project, ProjectRequest  # noqa: F401
 
 
 warn(
     f"The module {__name__} is deprecated and will be removed in version 4.17, "
-    "`Project` should be either imported from `ocp_resources.project_project_openshift_io` or "
+    "`Project` and `ProjectRequest` should be either imported from `ocp_resources.project_project_openshift_io` or "
```

Comment:
`ProjectRequest` only exists in `ocp_resources.project_project_openshift_io` group; please rephrase